modules/search/class.jetpack-search.php

```diff
@@ -490,6 +513,11 @@ public function search( array $es_args ) {
 		if ( is_wp_error( $request ) ) {
 			return $request;
 		}
+		$response_code = wp_remote_retrieve_response_code( $request );
```

Comment:
Looks like this line is duplicated down on 522, so the $response_code checking block should actually happen further down.

Reply:
done. nice catch